### DISCUSSION TITLE
Lade- Entladeplan zur Stromnetzentlastung

### DISCUSSION THREAD

**Discussion by rainer667** [hoylabs/OpenDTU-OnBattery#1205] [Ideas]
Hallo liebe Entwickler,
nach dem ich lauter doofe Ideen habe, wollte ich fragen, ob es möglich wäre eine Art von Sommer- Winterladeplan zu hinterlegen.
Die Idee dahinter ist, morgens in der schwachen Zeit das Netz zu unterstützen und z. B. erst ab Mittag den Speicher zu laden und dann nichts oder weniger einzuspeisen. Über den Victron wird, wenn SoC der Batterie erreicht ist, der Überschuss eingespeist.
Im Winterplan wird weiter wie gehabt geladen um den Akku möglichst schnell vollzuladen. Wer den Sommerplan nicht möchte, kann den Winterplan benutzen. 😉 Ich habe alles, was ich mit den Shellys verschieben kann, wie z.B. Boiler aufheizen und E-Bike Akku laden, schon auf den Nachmittag gelegt, weil der Speicher schon voll ist.  
Ich würde gerne den Sommerplan z.b. von Mai bis September hinterlegen weil ab ca. Mai der Speicher Mittags bereits voll ist. Das könnte unter den Dynamic Power Limiter Einstellungen, in Batterie SoC hinterlegt werden.
Um in den Sommerplan und einen leeren Speicher vor zu laden und zu unterstützen könnte evtl. ein fester Wert (z.b. 50% oder 200 Watt Ladeleistung) zum Vorladen hinterlegt werden, also Strom teilen am Vormittags wäre evtl. auch eine gute Option. 
Vielen Dank Rainer

**Comment by Snoopy-HSS** [comment]
Hi Rainer.

Mein letzter Stand ist, das aktuell in Deutschland nicht aus einer Batterie ins Netz eingespeist werden darf.
Du könntest ja sonst auch über Tiber Strom für 2-3c einkaufen, die Batterie laden und dann als Solarstrom wieder einspeisen und Geld dafür bekommen, das wäre dann Subventionsbetrug.

Es gibt eine Regelung, wenn die Batterie an einem eigenen zähler hängt...

**Reply by spcqike** [reply]
gibt es dazu was schriftliches? Ich dachte bisher, man darf das. IMmerhin wäre es sehr netzdienlich überschüssigen Strom zu kaufen, den akku zu verschleißen und dann wieder einzuspeisen.

ohne direktvermarktung muss der börsenstrompreis ja auch < -7cent/kWh sein, damit er nach Aufschlag der Steuer und Netzentgelte immernoch < 8Cent ist. mehr als 8Cent/kWh beim Einspeisen bekommt man ja nicht.

zum Thema:
wenn man eine entsprechende Anlage hat und die 8Cent/kWh  EInspeisevergütung bekommt, finde ich die Idee durchaus gut. Den Akku erst nach der börslichen Peakzeit (~ 07-09 Uhr) aufzuladen, solange er dann noch voll wird um den nächtlichen Eigenbedarf zu decken. die meisten Leute haben wohl aber ein BKW, wo es keine Vergütung gibt, oder nutzen ohnehin 99% des Stroms selbst. oder?

**Reply by rainer667** [reply]
Hallo @Snoopy-HSS,
nicht aus dem Akku ins Stromnetz laden. Anstatt den Akku schon morgens ab 8:00 bis 12:00 Uhr von 20% auf 80% zu laden, den mit den Panelen erzeugten Strom ins Stromnetz direkt einspeisen und mit dem Laden des Akkus erst z.B. 12:00 Uhr beginnen. 
Wenn Mittags die Panele (z.B. bei mir 3KW laden) ist der Speicher in ca. 2 Stunden voll und wenn es bewölkt ist dauerts dann 4 Stunden. Aber bis Abend wird er immer voll.
@spcqike mir gehts nicht um Geld, da gibts Nachmittag mehr zu verdienen, aber damit könnte man das Netz entlasten, wenn jeder der möchte z.B. bis 10:00 Uhr ins Stromnetz läd oder wie oben erwähnt teilt.
@spcqike es betrifft mit Sicherheit nur Personen die einen Speicher laden und das sind einige User.

**Reply by Snoopy-HSS** [reply]
Das wäre denkbar.

Dann ist es wohl ein angemeldetes 600W BKW, da ja nach der neuen Regelung zwar 800W aber nur noch 2kW PV Generator erlaubt ist ;-)

Ich Persönlich mit meinen 4x420W bin erst mal froh wenn die Batterie voll und Speise dann den Rest ein, nicht netzfreundlich, aber me first...
Und wie besprieben aber Herbst/Winter/Frühjahr ist man für jedes Watt in der Batterie froh...

Es gab tatsächlich da mal Vorschläge/Gedanken zur Netzentlastung, aber da wurde dann klar das eine Vergütung >20c notwendig ist um wirklich Investoren oder Homeuser dazu zu bewegen dies auch zu tun. Aber wie immer bei unserer Regierung ist das erst mal eingeschlafen.

Aber wir entlasten das Netz ja schon Nachts durch den Eigenverbrauch massiv...

**Reply by rainer667** [reply]
@Snoopy-HSS Ich habe 2 angemeldete Anlagen. 5kv auf dem Garagendach und die 2. Anlage mit 8 Modulen auf dem Balkon, die den Akku laden und die Nulleinspeisung über den DIY 51,2V 200Ah Akku, Victron Smart Shunt und Victron 250/100. Angemeldet sind 2 HM 1500 wobei eigentlich nur einer ins Netz einspeist wenn der Akku 80% voll ist. Den zweiten schalte ich am Nachmittag von Hand zu wenn ich dran denke :-)
Im Sommer könntest du sicherlich auch bis Mittags etwas Strom teilen und erst gegen Mittag zu 100% laden?

**Reply by spcqike** [reply]
> Im Sommer könntest du sicherlich auch bis Mittags etwas Strom teilen und erst gegen Mittag zu 100% laden? 

kommt halt stark auf die Ausrichtung der Module und Batteriekapazität an :) 

Wenn man sowas einbauen will, macht es Sinn, das an festen Zeiten und Plänen zu orientieren? Oder sollte man nicht eher den Markt / das Wetter hernehmen?

Es gibt APIs, wo man den Börsenstrompreis abfragen kann, auch wenn dieser nicht zwingend lokal gelten muss.

(Einschub: 
Soll heißen: nur weil der Börsenstrompreis grad bei 0 liegt, weil deutschlandweit zu viel Strom da ist, heißt das nicht, dass er das auch in meiner Region ist. Sauber betrachtet müsste ich meine Anlage dann abdrehen, wenn in meiner Region zu viel Strom vorhanden ist. Nutzt ja nichts, in Bayern die PV aus zu machen, weil im Norden Wind und SOnne ist, und dann gibt es doch den Redispatch und in Bayern wird das Gaskraftwerk hochgefahren. andersherum brauch ich nicht einspeisen, wenn bei mir bereits zu viel Strom ist der gar nicht abgenommen wird
Gibt es irgendwo solche Daten? Wo man sehen kann, was das Netz in der eigenen Region macht? )

für die Planung der eigenen Stromnutzung und damit der Entscheidung, ob und wenn ja bis wann die Akkuladung verschoben wird, könnte man PV Forecasts verwenden. Damit bekommt man schon mal eine grobe Tendenz, wie viel Energie am eigenen Standort zur jeweiligen Stunde mit der eigenen Anlage erzeugt werden kann. Ob die noch verfügbare Energie dann bis Abends reicht, um den Speicher voll zu kriegen und gleichzeitig die theoretische Grundlast abzudecken, kann man dann ja ausrechnen.
https://api.forecast.solar/estimate/52.61598/13.46571/4/-51/3.4

Damit braucht es kein Sommer/Winterplan und auch bei bewölkten Tagen würde der Speicher gleich zu beginn geladen.

**Reply by rainer667** [reply]
@spcqike Das wäre natülich das optimale Vorgehen. Mir würde ja schon die Möglichkeit Vormittags einzuspeisen gut gefallen. Im Moment muss immer erst der Akku  beim eingestellten SoC der Batterie sein, damit der Rest ins Netz geht. Mein SoC ist seit ca. 11:00  Uhr auf 80% und der eine HM 1500 speist ein. Ich hab gerade mal Nachgesehen und musste feststellen hab schon wieder 94% und deshalb den zweiten HM 1500 zuschalten musste...

**Reply by spcqike** [reply]
Theoretisch ist es eine einfache Betrachtung. Denke ich. 

Man muss wissen wie viel Energie der Speicher hat (die gesamt Kapazität) Wie voll er abends sein soll(Ziel vor der Nacht). Wie voll er ist (SoC, haben wir). Dazu die angenommene Konstante Grundlast (haben wir auch, oder?) 

die API gibt stündliche Energie Werte aus. Von jeder Stunde muss die Grundlast abgezogen werden. Das was über bleibt muss von hinten nach vorn addiert werden und geguckt werden, wann der Akku halt voll wäre. 

Zwischenzeitige Entladungen über die Grundlast hinaus, sind natürlich unbekannt und können nicht berücksichtigt werden. 

Irgendwie so. 

Muss man nur wen finden, der es einbauen kann und will :) 

die API verwende ich bereits seit ein paar Monaten mit node red um meinen WR per MQTT etwas zu steuern. Sie ändert die Stundenprognosen auch im laufe des Tages, wenn sich das Wetter doch anders verhalten sollte als geplant. Sie würde also auch ein ungeplantes Sommergewitter im laufe des Tages berücksichtigen und den forecast reduzieren. Wie viel Vorlauf da existiert, habe ich aber nicht betrachtet. Mir ist halt nur aufgefallen dass die zu erwartbaren tageserträge sich mehrfach nach oben korrigiert haben, als es doch sonniger war als gedacht.

**Reply by rainer667** [reply]
Danke für die schönen Gedanken zu diesen Thema, leider wirds wohl nix. 
Wenn es um was geht was nur der Allgemeinheit hilft, also ohne persöhnlichen Vorteil, hat niemand interesse.

**Reply by spcqike** [reply]
> Wenn es um was geht was nur der Allgemeinheit hilft, also ohne persöhnlichen Vorteil, hat niemand interesse.

das sehe ich nicht so. und ich hoffe auch, dass sich das zukünftig ändert, aber das obliegt eher der Politik. und dann wirds wieder für das persönliche Interesse :) 

Auf der anderen Seite ist auch niemandem (von uns) geholfen, wenn die Spannung im Netz aufgrund von Stromüberschüssen so hoch wird, dass die Wechselrichter anfangen sich zu drosseln oder gar den Betrieb einzustellen. Das Verständnis muss nur ankommen, dann sehen auch mehr Leute einen Sinn dahinter, den Akku nicht von 8-10 zu laden sondern eben erst von 12-13:15 :)


Und zum Thema: die Umsetzung ist, auch wenn ich sie blauäugig und leicht beschrieben habe, wahrscheinlich nicht so trivial 😄 mit Node-Red würde ich mir sowas wahrscheinlich in ein paar Tagen zusammen "ChatGPTen", aber es dann in eine Firmware auf dem ESP zu bekommen und das vorhandene System dabei nicht zu beeinträchtigen, macht es schon wieder komplexer.

**Reply by rainer667** [reply]
Leider traurig aber wahr!!
Das es nicht einfach wird, kann ich nachvollziehen. War auch nur ein doofer Gedanke von mir. Ich spinne diese Gedanken seit ca. 26 Jahren wo ich begonnen habe mein Passivhaus zu bauen :-)

**Comment by Snoopy-HSS** [comment]
Aber warum denn morgens Strom aus der Batterie vom BKW verschenken, wenn ich nicht weiß, ob ich es Nachmittags wieder voll bekomme...
Würde ja nur bei alten Zählern Sinn machen die rückwärts laufen. Und deren Tage sind mittlerweile gezählt.

Gesetz für den Ausbau erneuerbarer Energien (Erneuerbare-Energien-Gesetz - EEG 2023)
§ 21 Einspeisevergütung und Mieterstromzuschlag

Siehe Absatz (2).2

(2) Anlagenbetreiber, die die Einspeisevergütung in Anspruch nehmen,

1.    müssen dem Netzbetreiber den gesamten in dieser Anlage erzeugten Strom zur Verfügung stellen, der

    a)        nicht in unmittelbarer räumlicher Nähe zur Anlage verbraucht wird und
    b)        durch ein Netz durchgeleitet wird, und

**2.    dürfen mit dieser Anlage nicht am Regelenergiemarkt teilnehmen.**

**Reply by spcqike** [reply]
> warum denn morgens Strom aus der Batterie vom BKW verschenken

So hab ich es nicht verstanden 

> morgens in der schwachen Zeit das Netz zu unterstützen und z. B. erst ab Mittag den Speicher zu laden und dann nichts oder weniger einzuspeisen.

ich verstehe den Wunsch so, dass vormittags Überschüsse eingespeist werden und erst ab mittags in die Batterie gehen. 

Vormittags für Umme aus meinem Akku einspeisen? Ne das wäre blöd. Auch wenn es sicher netzdienlich und steuerbar (im Sinne von der Akku wird wieder voll) wäre, wäre mir der Akku dafür zu schade, bei den 8Cent. 

auch finde ich die Idee des netzeinspeisung reduzieren wenn zu viel Strom auf den Markt ist gut. Das hab ich in der Vergangenheit schon irgendwo angesprochen für reine solar Systeme (ohne Akku). In Verbindung mit bspw tibber um den börsenpreis als Indiz der netzauslastung zu nehmen. 


Den Paragraphen gucke ich mir mal an 👍

**Reply by spcqike** [reply]
zum Gesetz:

Bist du sicher, dass eine heimische Anlage überhaupt in die Kategorie "Regelenergie" kommt? und damit überhaupt am Regelenergiemarkt teilnehmen kann?
https://www.bundesnetzagentur.de/DE/Fachthemen/ElektrizitaetundGas/Versorgungssicherheit/Netzengpassmanagement/Engpassmanagement/Regelenergie/start.html

ich bin immer noch der Meinung, dass man mit einem Akku durchaus einspeisen kann und darf und die üblichen 8Cent/kWh bekommt. Das ist noch keine Regelenergie, auf die der Netzbetreiber zugriff hätte (und als echte Regelenergie auch haben muss).

und ja, da hat das Gesetzt auch Recht, eine Einspeisende Anlage kann keine Regelenergieanlage sein. Regelenergie greift ja kurzfristig und mittelfristig, dient ganz anderen Zwecken und bekommt dafür eine andere Vergütung. 

hier (https://www.netzfrequenzmessung.de/index.htm) sieht man auch eine Live Anzeige zur Netzfrequenz und zur Primärregelenergie. Allein dafür, dass sie erfasst und extern gesteuert werden kann, müssen ganz andere Voraussetzungen geschaffen sein, als für eine einfache Einspeisung.

nachtrag:
scheinbar kann man als Teil eines virtuellen Kraftwerks zur Regelenergie beitragen (https://energyload.eu/stromspeicher/solarstromspeicher/heimspeicher-regelenergiemarkt/), aber dann speist man halt auch nicht mehr einfach ein.
